tests/runner/pom.xml

```diff
@@ -27,6 +27,31 @@
             <artifactId>aether-impl</artifactId>
             <version>${aetherVersion}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
```

Comment:
Those dependencies were required before this PR but were included in the classpath because another dependency, that does not depend on this anymore, was using them.